src/state/reducers/userReducer.js

```diff
@@ -1,14 +1,26 @@
 // export const USER_EVENT_START = 'USER_EVENT_START';
-import { USER_PROFILE } from '../actions/index';
+import * as a from '../actions/index';
```

Comment:
Just a pet peeve of mine, but could we rename `a` to `ACTIONS` or something? Single-letter variables can get difficult to keep track of. Especially down the road when our redux store is significantly larger than it is now.